common/lib/util/utils.js

```diff
@@ -129,10 +129,16 @@ var Utils = (function() {
 	 * of another constructor
 	 * See node.js util.inherits
 	 */
-	Utils.inherits = (typeof(require) === 'function' && require('util') && require('util').inherits) || function(ctor, superCtor) {
+
+	Utils.inherits = function(ctor, superCtor) {
 		ctor.super_ = superCtor;
 		ctor.prototype = Utils.prototypicalClone(superCtor.prototype, { constructor: ctor });
 	};
+	try {
+        if (typeof(require) === 'function' && require('util') && require('util').inherits) {
+        	Utils.inherits = require('util').inherits;
+		}
+    } catch (err) {  /* Do Nothing. */ }
```

Comment:
Other than don't we need to change the platforms on all of them, Browser, encryption, node, react, and nativescript.   That seems out of place for this pull request.